packages/dev-server/index-worker.ts

```diff
@@ -2,6 +2,10 @@ import { bootstrapWorker } from '@vendure/core';
 
 import { devConfig } from './dev-config';
 
+// https://github.com/vendure-ecommerce/vendure/issues/152
+// fix race condition when modifying DB
+devConfig.dbConnectionOptions = { ...devConfig.dbConnectionOptions, synchronize: false };
```

Comment:
This is the right idea, but putting it here only solves the issue for those people working on the Vendure repo itself. If you were to install the @vendure/core npm package, you would still run into the issue.

I suggest that the fix be put here: https://github.com/vendure-ecommerce/vendure/blob/master/packages/core/src/bootstrap.ts#L100. That way it fixes the issue for everyone, no matter if they are developing on the Vendure repo or using the npm package.